docs/README.md

```diff
@@ -59,6 +58,16 @@ The credentials can be supplied by providing the `modules.core.aws` setting in t
 }
 ```
 
+AWS SDK can be also configured in code by using `altis.aws_sdk.params` filter which is called right before the global instance of the AWS SDK is created.
+
+```
+add_filter( 'altis.aws_sdk.params', 'aws_sdk_params' );
+
+function aws_sdk_params( $params ) {
+	return $params['region'] = 'new_region';
```

Comment:
This code isn't going to work, it'll just return the region rather than the array.